src/linear_algebra/basic.lean

```diff
@@ -273,6 +273,8 @@ end
 lemma coe_pow (f : M →ₗ[R] M) (n : ℕ) : ⇑(f^n) = (f^[n]) :=
 by { ext m, apply pow_apply, }
 
+@[simp] lemma id_pow (n : ℕ) : (id : module.End R M)^n = id := one_pow n
```

Comment:
I don't know if it matters, but I'd be inclined not to use `End` here, since none of the other lemmas do:
```suggestion
@[simp] lemma id_pow (n : ℕ) : (id : M →ₗ[R] M)^n = id := one_pow n
```

Reply:
Good point, dropping `module.End`.